engine/dlib/src/remotery/lib/Remotery.c

```diff
@@ -59,14 +59,6 @@
 #define RMT_IMPL
 #include "Remotery.h"
 
-// DEFOLD
-#include <dmsdk/dlib/log.h>
-#include <dmsdk/dlib/hash.h>
-
-static const char* g_EmptyString = "<empty>"; // As seen in profile_remotery.cpp _rmt_HashString32()
-static rmtU32 g_EmptyHash = 0;
-// END DEFOLD
-
```

Comment:
Granted, this workaround shouldn't have been needed in the first place.
Just make a mental note that we've removed this now.